docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -65,109 +66,100 @@ To connect the AWS Distro to the AWS services, create a Secret containing the cr
-         log_stream_template \$kubernetes['namespace_name'].\$kubernetes['pod_name'].\$kubernetes['container_name']
-         log_stream_name from-kyma-cluster
-         log_retention_days 7
-     variables:
-       - name: AWS_ACCESS_KEY_ID
-         valueFrom:
-           secretKeyRef:
-             name: aws-credentials
-             namespace: $K8S_NAMESPACE
-             key: AWS_ACCESS_KEY_ID
-       - name: AWS_SECRET_ACCESS_KEY
-         valueFrom:
-           secretKeyRef:
-             name: aws-credentials
-             namespace: $K8S_NAMESPACE
-             key: AWS_SECRET_ACCESS_KEY
-       - name: AWS_REGION
-         valueFrom:
-           secretKeyRef:
-             name: aws-credentials
-             namespace: $K8S_NAMESPACE
-             key: AWS_REGION
    EOF
    ```
 
-2. Deploy a TracePipeline:
+2. Deploy a [TracePipeline](./../../03-traces.md):
 
    ```bash
    kubectl apply -f - <<EOF
    apiVersion: telemetry.kyma-project.io/v1alpha1
    kind: TracePipeline
    metadata:
-     name: aws-xray
+     name: aws
    spec:
      output:
        otlp:
          endpoint:
-           value: http://otel-collector.$K8S_NAMESPACE.svc.cluster.local:4317
+           value: http://$HELM_OTEL_AWS_RELEASE-opentelemetry-collector.$K8S_NAMESPACE.svc.cluster.local:4317
    EOF
    ```
 
-3. Deploy a MetricPipeline:
+3. Deploy a [MetricPipeline](./../../04-metrics.md):
 
    ```bash
    kubectl apply -f - <<EOF
    apiVersion: telemetry.kyma-project.io/v1alpha1
    kind: MetricPipeline
    metadata:
-     name: aws-cloudwatch
+     name: awsh
    spec:
      input:
        runtime:
          enabled: true
-       istio:
-         enabled: true
-       prometheus:
-         enabled: true
      output:
        otlp:
          endpoint:
-           value: http://otel-collector.$K8S_NAMESPACE.svc.cluster.local:4317
+           value: http://$HELM_OTEL_AWS_RELEASE-opentelemetry-collector.$K8S_NAMESPACE.svc.cluster.local:4317
    EOF
    ```
 
 ## Verify the Results
 
-Verify that the logs and metrics are exported to CloudWatch and that the traces are exported to X-Ray.
+Verify that the logs, traces and metrics are exported to CloudWatch.
 
 1. [Install the OpenTelemetry demo application](../opentelemetry-demo/README.md).
 2. Go to `https://{AWS_REGION}.console.aws.amazon.com/cloudwatch`. Replace `{AWS_REGION}` with the region that you have chosen when [creating the Secret with AWS credentials](#create-a-secret-with-aws-credentials).
-3. To verify the traces: under **X-Ray traces**, go to **Traces**.
-4. To verify the logs: under **Logs**, go to **Log groups** and select the log group of your cluster which has a name that follows the pattern `/logs/{CLUSTER_IDENTIFIER}`. Now, you can open the log stream you want and view the logs. The name of each log stream follows the pattern `{NAMESPACE}.{POD_NAME}.{CONTAINER_NAME}`.
+3. To verify the traces: under **Apllication Signals**, go to **Traces**.
```

Comment:
```suggestion
3. To verify the traces: under **Application Signals**, go to **Traces**.
```
Disregard this suggestion in case AWS truly uses "Apllication" for the UI element.